lib/deployer.js

```diff
@@ -48,10 +48,10 @@ module.exports = function(args) {
   }
 
   function git() {
-    var len = arguments.length;
-    var args = new Array(len);
+    const len = arguments.length;
```

Comment:
I propose to convert to rest parameters.

Reply:
I refactored to spread, I'll change it to rest.

Reply:
I do not know why you expand arguments object with spread syntax without using rest parameters for git.
I would be glad if you could answer the reason :)

Reply:
indeed, it should be [rest](https://stackoverflow.com/questions/960866/how-can-i-convert-the-arguments-object-to-an-array-in-javascript).